### PR TITLE
rpc: make max_body_bytes and max_header_bytes configurable

### PR DESCRIPTION
**Pull request by bluele**
#3817

* [x] Referenced an issue explaining the need for the change
* [x] Updated all relevant documentation in docs
* [x] Updated all code comments where relevant
* [ ] Wrote tests
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by marbar3778** [comment]
Can you please add a changelog_pending entry, thank you

**Comment by bluele** [comment]
@marbar3778 
I updated CHANGELOG_PENDING.md.

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ program](https://hackerone.com/tendermint).
 ### BREAKING CHANGES:
 
 - CLI/RPC/Config
+- [rpc] \#3818 Make `max_body_bytes` and `max_header_bytes` configurable
```
Comment (marbar3778) [inline CHANGELOG_PENDING.md]: Can you move this to Improvements, it is not breaking anything.

Comment (bluele) [reply CHANGELOG_PENDING.md]: @marbar3778 
oh, sorry. I fixed it.

**Pull request merged**